src/react/deckgl.js

```diff
@@ -55,7 +56,8 @@ const defaultProps = {
   onWebGLInitialized: noop,
   onAfterRender: noop,
   onLayerClick: null,
-  onLayerHover: null
+  onLayerHover: null,
+  useDevicePixelRatio: false
```

Comment:
Good question. My take: Unless the default changed between 4.0 and 4.1 (in which case we could consider it a bug), let's keep it unchanged for now (`true`) and decide in review based on performance data. 

This PR is intended for a patch release after all, and is already pushing it by adding a new prop.